packages/stories/src/gitgraph-js/6-custom-renders.stories.tsx

```diff
@@ -300,4 +306,235 @@ storiesOf("gitgraph-js/6. Custom renders", module)
         }}
       </GraphContainer>
     );
+  })
+  .add("custom renderer - message below tags", () => {
+    // Providing a custom Renderer class can enable limitless customization.
+    // However -
+    // NOTE: using a custom Renderer can be considered advanced and experimental.
+    //   1. You will have to read the code to figure out which methods to override.
+    //   2. The internal API (ie. the Renderer methods and variables) is *internal* and
```

Comment:
👏 For the docs along this use-case. I indeed think this should be considered experimental until we have a few more iterations on how it goes.